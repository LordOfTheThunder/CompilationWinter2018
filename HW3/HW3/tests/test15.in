void TestAssignByteToInt() {
	int x = 33b;
}




void main() { print("I hate this course"); }



