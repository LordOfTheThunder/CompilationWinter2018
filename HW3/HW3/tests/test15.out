---end scope---
x INT 0
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
TestAssignByteToInt ()->VOID 3
